.eslintignore

```diff
@@ -15,3 +15,4 @@
 *.testjs
 *.testts
 *__snapshots__/*
+*.ts
```

Comment:
uh, why is *.ts not being linted? this is the main point of the linter...

Reply:
iirc TSLint may be handling linting but I may be wrong.

Reply:
If I remove `*.ts` from this eslintignore then I get errors about missing extensions in my import statements (`import/extensions`) which doesn't seem be fixable by configuring eslint.

Reply:
you should set `'import/extensions': 0,` instead of completely disabling the linter lol

Reply:
I think I was trying to set it to 'never'. I'll try `0`